packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/templates/expandableSearchBoxInput.html

```diff
@@ -1,20 +1,21 @@
-<!-- sldsValidatorIgnore -->
 <template>
   <div class="slds-form-element z-index-high searchbox__container-wrapper">
     <div class="slds-form-element__control searchbox_floating-container">
       <div class="slds-combobox_container">
         <div
           class="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click"
-          aria-expanded="false"
+          aria-expanded="true"
```

Comment:
This should start false, and it should be updated to true only when the suggestions are visible. This is accessibility requirements.